source/unit/Fixture.ooc

```diff
@@ -170,9 +170,10 @@ Fixture: abstract class {
 }
 
 TestFailedException: class extends Exception {
+	description: String
```

Comment:
It worked some of the time, but not always.
An idea is to have a `_literal` member in `String` which defaults to `false`, and only `makeStringLiteral(...)` sets it to `true`, and then a corresponding to property to use. Don't know how useful it is (or if there are cases where this isn't good).